processor/issueProcessor/opened.go

```diff
@@ -31,10 +31,27 @@ import (
 func (ip *IssueProcessor) ActToIssueOpened(issue *github.Issue) error {
 	ip.attachLabels(issue)
 	ip.attachComments(issue)
-
+	ip.autoTranslate(issue)
 	return nil
 }
 
+func (ip *IssueProcessor) autoTranslate(issue *github.Issue) error {
+	translateTitle := ip.Translator.Translate(*issue.Title)
+	translateBody := ip.Translator.Translate(*issue.Body)
+	if translateTitle == "" && translateBody == "" {
```

Comment:
Please double check here if the logic should be like the following:
```
if translateTitle == "" || translateBody == "" {
     return nil
}
```
@Frankzhaopku

Reply:
This is right, because either title or body need to be translated, we need to edit the issue, maybe the first edition is wrong.